setup.py

```diff
@@ -48,7 +48,10 @@ def getVersionNumber():
     url="https://github.com/rs-station/rs-booster",
     project_urls=PROJECT_URLS,
     python_requires=">3.7",
-    install_requires=["reciprocalspaceship", "matplotlib", "seaborn"],
+    install_requires=["reciprocalspaceship",
+                      "matplotlib",
+                      "seaborn==0.11.2"
```

Comment:
Let's make this "<=" -- I don't think this needs to be pinned to a specific version